lte/gateway/c/core/oai/tasks/nas5g/src/M5GPDUSessionModificationCommandReject.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+   Copyright 2020 The Magma Authors.
+   This source code is licensed under the BSD-style license found in the
+   LICENSE file in the root directory of this source tree.
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+#include <sstream>
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "lte/gateway/c/core/oai/common/log.h"
+#ifdef __cplusplus
+}
+#endif
+#include "lte/gateway/c/core/oai/tasks/nas5g/include/M5GPDUSessionModificationCommandReject.h"
+#include "lte/gateway/c/core/oai/tasks/nas5g/include/M5GCommonDefs.h"
+#include "lte/gateway/c/core/oai/tasks/nas5g/include/M5gNasMessage.h"
+
+namespace magma5g {
+PDUSessionModificationCommandReject::PDUSessionModificationCommandReject() {}
+PDUSessionModificationCommandReject::~PDUSessionModificationCommandReject() {}
+
+int PDUSessionModificationCommandReject::
+    EncodePDUSessionModificationCommandReject(
+        PDUSessionModificationCommandReject* pdu_sess_mod_comd_rej,
```

Comment:
Can we enable this Encode function only for UT, as the functional code do not make use of it. Similarly here, we have some more Encode functions which are used only for UT

Reply:
UT Already enabled for this